src/applications/vaos/components/FacilityPhone.jsx

```diff
@@ -8,5 +8,14 @@ export default function FacilityPhone({ contact }) {
 
   const [number, extension] = contact.split('x');
 
-  return <Telephone contact={number} extension={extension} />;
+  return (
+    <>
+      <Telephone contact={number} extension={extension} />
+      <br />
+      <h4 className="vads-u-font-family--sans vads-u-display--inline vads-u-font-size--base">
```

Comment:
I agree wit @jbalboni comment about skipping heading levels here. I took a look with the `headingsMap` plugin to confirm. I noticed the H4's are 1.6rem font-size, and the H2's are 1.5rem font-size.

---
![Screen Shot 2021-03-29 at 12 03 53 PM](https://user-images.githubusercontent.com/934879/112873600-769e4280-9087-11eb-9246-88e616e35e7a.png)